integration-tests/templates/react-native-test-app-template/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-native-test-app-template",
+  "name": "rn74-template",
```

Comment:
shoult the name of the directory be `rn74-template` as per this name?

Reply:
updated